app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_years,dog_years,result",
+    [
+        pytest.param(15, 15, [1, 1], id="exactly one year"),
+        pytest.param(24, 24, [2, 2], id="exactly two years"),
+        pytest.param(28, 29, [3, 3], id="exactly three years"),
+        pytest.param(32, 34, [4, 4], id="exactly four years"),
+        pytest.param(0, 0, [0, 0], id="zero years"),
+        pytest.param(7, 14, [0, 0], id="zero years - we discard the reminder"),
+        pytest.param(15, 20, [1, 1], id="1 year, we discard the reminder"),
```

Comment:
![image](https://user-images.githubusercontent.com/104218550/221560895-dfdfd331-8143-4b90-a792-cee1d6b290f8.png)

No need to use the same values, but it's better to add checks for all age boundaries for both(cat and dog). For example, you check that 14 gives 0 in the result only for a dog, but what about a cat.  Also, make more descriptive ids